sdk/storage/azure-storage-file-datalake/src/main/java/com/azure/storage/file/datalake/DataLakePathAsyncClient.java

```diff
@@ -635,6 +649,260 @@ public DataLakeServiceVersion getServiceVersion() {
                 response.getDeserializedHeaders().getLastModified())));
     }
 
+    /**
+     * Sets the access control on a path and subpaths.
```

Comment:
Can we make this something like Recursively sets access control on a path and its subpaths, and so on? Right now it could read as just 2 levels

Reply:
Ya